localstack/services/awslambda/event_source_listeners/sqs_event_source_listener.py

```diff
@@ -0,0 +1,186 @@
+import time
+from typing import Any, Dict, List
+
+from localstack.services.awslambda import lambda_executors
+from localstack.services.awslambda.event_source_listeners.event_source_listener import (
+    EventSourceListener,
+)
+from localstack.services.awslambda.lambda_api import (
+    LOG,
```

Comment:
should we use a separate logger for this file? importing loggers from other files is not something we typically do

Reply:
good catch